src/app/services/weather.service.ts

```diff
@@ -9,7 +9,7 @@ import { WeatherAlertResponse } from '../models/weather-alert.model';
   providedIn: 'root'
 })
 export class WeatherService {
-  private serverURL: string = 'https://api.weather.gov/alerts/active';
+  private serverURL: string = 'https://api.weather.gov/alerts';
```

Comment:
this needs set back to `/active` as this will break the current set up in master

Reply:
Fixed